df.md

```diff
@@ -0,0 +1,49 @@
+# 🦈 Senior Product Developer - Data Farming 
+[Ocean Protocol](https://www.oceanprotocol.com) builds tools for people to reclaim control of their data and unleash an open data economy. Ocean tools make it easy to publish & consume data services as data access tokens, and bridge data to DeFi. 
+
+Ocean has a vibrant, expanding ecosystem of data suppliers, data consumers, stakers, and community developers. To serve it better, we’re hiring! If you are someone that loves to build things and cares deeply about the future of data, we want to hear from you.
```

Comment:
We can make the above more specific.

How about:

Data Farming is a specific initiative in the spirit of liquidity mining, with generous OCEAN rewards to ecosystem participants who drive the supply of data. We're growing the Data Farming team. If you love to work with crypto, to build and to engage deeply with an enthusiastic community, we want to hear from you.